### PR TITLE
[DS-424]: [Pyxis React] Expose all the interfaces used to type props

### PR DESCRIPTION
**Pull request by emanuelemacri**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-424

## ✨ Enhancements
Export internal component types

### CODE REVIEW

File: packages/pyxis-react/src/components/Button/index.ts [hunk 1]
```diff
@@ -1,2 +1,4 @@
 export { Button } from './Button';
-export type { ButtonProps } from './types';
+export type {
+  ButtonProps, ButtonSize, ButtonVariant, ButtonContentProps, ButtonIconPlacement,
```
Comment (lauratrapani) [inline packages/pyxis-react/src/components/Button/index.ts:3]: Is necessary to export `ButtonContentProp`? It should be used only in an non-exposed component

File: packages/pyxis-react/src/components/Grid/Col/index.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
 export { Col } from './Col';
-export type { ColProps } from './Col';
+export type { ColProps, GridColSpan } from './Col';
```
Comment (lauratrapani) [inline packages/pyxis-react/src/components/Grid/Col/index.ts:2]: If the first is `ColProps` I think that also the second should be just `ColSpan`

File: packages/pyxis-react/src/components/Grid/Row/Row.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { ColProps } from 'components/Grid/Col';
 
 const getClasses = (
   size?: GridRowSize,
-  columnsFrom?: RowBreakpoint,
+  columnsFrom?: GridRowBreakpoint,
```
Comment (lauratrapani) [inline packages/pyxis-react/src/components/Grid/Row/Row.tsx:7]: The same as the `Col`

**Review by Gsciarra — DISMISSED** [review]

File: packages/pyxis-react/src/components/TextSwitch/TextSwitch.tsx [hunk 4]
```diff
@@ -87,14 +87,14 @@ export const TextSwitch: FC<TextSwitchProps> = ({
 
 export type TextSwitchLabelPosition = 'topCenter' | 'topLeft' | 'left';
 
-type TextSwitchOptionWidth = 'equal' | 'inherit';
+export type TextSwitchOptionWidth = 'equal' | 'inherit';
 
-interface TextSwitchSliderPosition {
+export interface TextSwitchSliderPosition {
```
Comment (lauratrapani) [inline packages/pyxis-react/src/components/TextSwitch/TextSwitch.tsx:92]: This is just used in the `useState`, so it should be internal

File: packages/pyxis-react/src/internalComponents/ControlGroup/index.ts [hunk 5]
```diff
@@ -2,6 +2,12 @@ export { ControlGroup } from './ControlGroup';
 export { CardOption } from './CardOption';
 export { Option } from './Option';
 
-export type { ControlGroupProps } from './ControlGroup';
+export type {
```
Comment (lauratrapani) [inline packages/pyxis-react/src/internalComponents/ControlGroup/index.ts:5]: Maybe it's better to export them with a proper alias from `RadioGroup`, `RadioCardGroup` and so on

**Review by lauratrapani — CHANGES_REQUESTED** [review]
Just small things

**Review by Gsciarra — APPROVED** [review]

**Review by lauratrapani — APPROVED** [review]

**Pull request merged**